src/Audio/Listener.cs

```diff
@@ -69,28 +69,28 @@ public static Vector3f UpVector
         }
         
         #region Imports
-        [DllImport("csfml-audio-2", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Comment:
https://msdn.microsoft.com/en-us/library/system.security.suppressunmanagedcodesecurityattribute(v=vs.110).aspx

It makes .NET not perform a stack walk when calling a native function.
Direct quote from the link: "The stack walk performed when calling unmanaged code is omitted at run time, resulting in substantial performance savings."